x11-packages/nextcloudcmd/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nextcloud/desktop
+TERMUX_PKG_DESCRIPTION="Command-line client tool for Nextcloud."
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Zhymabek Roman <robanokssamit@yandex.ru>"
+TERMUX_PKG_VERSION=3.12.3
+TERMUX_PKG_REVISION=1
```

Comment:
Since this is the initial packaging of nextcloud-client, please remove the `TERMUX_PKG_REVISION=1` line.
This is only used when rebuilding a package without a version change.
```suggestion

```

Reply:
Delete package revision version